src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -201,7 +201,9 @@
       <PostDeploymentScriptArgument>@(PostDeploy->'--postdeploy %(Identity)', ' ')</PostDeploymentScriptArgument>
       <RefactorLogScriptArgument>@(RefactorLog->'--refactorlog %(Identity)', ' ')</RefactorLogScriptArgument>
       <DebugArgument Condition="'$(MSBuildSdkSqlProjDebug)' == 'True'">--debug</DebugArgument>
-      <DacpacToolCommand>dotnet &quot;$(DacpacToolExe)&quot; build $(OutputPathArgument) $(MetadataArguments) $(SqlServerVersionArgument) $(InputFileArguments) $(ReferenceArguments) $(SqlCmdVariableArguments) $(PropertyArguments) $(PreDeploymentScriptArgument) $(PostDeploymentScriptArgument) $(RefactorLogScriptArgument) $(DebugArgument)</DacpacToolCommand>
+      <TreatTSqlWarningsAsErrorsArgument  Condition="'$(TreatTSqlWarningsAsErrors)' == 'True'">--warnaserror</TreatTSqlWarningsAsErrorsArgument>
```

Comment:
If we add an `Or '$(TreatWarningsAsErrors)' == 'True'` to the `Condition` here we can help users used to C# projects to use that property which has the same semantic meaning.